app/src/ui/diff/diff-helpers.tsx

```diff
@@ -352,3 +353,45 @@ export function canSelect(
 ): file is WorkingDirectoryFileChange {
   return file instanceof WorkingDirectoryFileChange
 }
+
+/** Gets the width in pixels of the diff line number gutter based on the number of digits in the number */
+export function getLineWidthFromDigitCount(digitAmount: number): number {
+  let diffSize: number
+
+  if (digitAmount <= 3) {
+    diffSize = 35
+  } else {
+    diffSize = 10 * digitAmount + 5
+  }
+
+  return diffSize
```

Comment:
Minor Suggestion - We try to make use of constants where possible and follow a pattern of tertiary operators to do so. Also at top of method you could have a `const minLineWidth = 35` to make it just a bit easier to read at a glance. Thus this function could be something like `return digitAmount <= 3 ? minLineWidth : 10 * digitAmount + 5`.

Reply:
or…
```typescript
return Math.max(digitAmount, 3) * 10 + 5
```